tests/monitors/kafka/kafka_test.py

```diff
@@ -1,112 +1,163 @@
 """
 Monitor tests for kafka
 """
-import textwrap
 from contextlib import contextmanager
 from functools import partial as p
 
 import pytest
+
 from tests.helpers.agent import Agent
 from tests.helpers.assertions import has_datapoint_with_dim, has_datapoint_with_metric_name, tcp_socket_open
-from tests.helpers.util import container_ip, run_container, run_service, wait_for
+from tests.helpers.metadata import Metadata
+from tests.helpers.util import (
+    container_ip,
+    run_container,
+    run_service,
+    wait_for,
+    DEFAULT_TIMEOUT,
+    random_hex,
+    container_hostname,
+)
+from tests.helpers.verify import verify
 
 pytestmark = [pytest.mark.collectd, pytest.mark.kafka, pytest.mark.monitor_with_endpoints]
+KAFKA_METADATA = Metadata.from_package("collectd/kafka")
+PRODUCER_METADATA = Metadata.from_package("collectd/kafkaproducer")
+CONSUMER_METADATA = Metadata.from_package("collectd/kafkaconsumer")
+VERSIONS = ["0.9.0.0", "0.10.0.0", "0.11.0.0", "1.0.0", "1.0.1", "1.1.1", "2.0.0"]
```

Comment:
Any basis for all these versions? 3 `1.x`s feel like redundant to me if there's no significant diffs.

Reply:
I have no idea, I just kept what was there.

Reply:
@asuresh4 any thoughts?

Reply:
Got no comment if that's the case. Thanks

Reply:
@seonsfx @jaysfx - can't remember what the rationale was when I added these tests, maybe there was none. Looking at the release notes for kafka, I think we should be ok without the test for `1.0.1`.